jetbrains-core/src/software/aws/toolkits/jetbrains/services/caws/CawsSpaceProjectInfo.kt

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.caws
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.popup.ListPopup
+import com.intellij.openapi.util.Disposer
+import com.intellij.openapi.wm.StatusBar
+import com.intellij.openapi.wm.StatusBarWidget
+import com.intellij.openapi.wm.StatusBarWidgetFactory
+import com.intellij.openapi.wm.impl.status.EditorBasedWidget
+import com.intellij.util.Consumer
+import java.awt.event.MouseEvent
+
+private const val WIDGET_ID = "CawsSpaceProjectInfo"
+
+class CawsStatusBarInstaller : StatusBarWidgetFactory {
```

Comment:
Is it possible to limit this status bar view to only display it in remote/thin clients?

Reply:
Yes, its showing only in remote/thin clients

Reply:
i'm not seeing any logic scoping this specificly to remote clients